### PR TITLE
check staff group rather than is_staff for elevated perms

### PR DESCRIPTION
**Pull request by escattone**
mozilla/sumo#1683

### CODE REVIEW

**Review by smithellis — APPROVED** [review]
I think you caught them all, @escattone !

File: kitsune/sumo/templatetags/jinja_helpers.py [hunk 1]
```diff
@@ -467,11 +465,6 @@ def static(path):
         return ""
 
 
-@library.global_function
```
Comment (akatsoulas) [inline kitsune/sumo/templatetags/jinja_helpers.py:470]: let's revert this, it's not relevant to this PR

Comment (akatsoulas) [reply kitsune/sumo/templatetags/jinja_helpers.py:470]: On second thought let's keep it to clean things up a bit

File: kitsune/sumo/templatetags/jinja_helpers.py [hunk 2]
```diff
@@ -565,9 +558,6 @@ def show_header_fx_download(context):
         return True
 
 
-@library.global_function
-def is_trusted_user(user):
-    return is_trusted_user_func(user)
-
-
+library.global_function(webpack_static)
```
Comment (akatsoulas) [inline kitsune/sumo/templatetags/jinja_helpers.py:561]: I would move all the Jinja function registrations to the top of the file

**Review by akatsoulas — APPROVED** [review]
Looks good to me! r+wc

**Comment by akatsoulas** [comment]
For completeness we could also remove the underscore from https://github.com/mozilla/kitsune/blob/main/kitsune/messages/forms.py#L24

**Pull request merged**